### ISSUE TITLE
State of some buttons not obvious

### ISSUE THREAD

**Issue by 0q98ahdsg3987y1h987y** [pcperini/Tusk---Issues#21]
Not a bug. Things like the favorite/like button, which changes from outline to filled in are ambiguous. Maybe consider making them blue (or if you’ll add themes: the primary tint color) when active.

**Comment by pcperini** [comment]
They should be! Ergo, I'm going to treat this as a bug.

**Comment by pcperini** [comment]
There have been a handful of fixes for this across the last few builds. Does this feel a little clearer now?

**Comment by 0q98ahdsg3987y1h987y** [comment]
yes, the yellow helps. maybe a little animation would be even better, but that's probably more something for point release polish
